### PR TITLE
Merge Community meeting #1 Notes

### PR DESCRIPTION
**Pull request by stefan-wuest**
Documentation of notes of Merge Community meeting #1 according to
https://github.com/ethereum-cat-herders/PM/issues/246

### CODE REVIEW

**Comment by stefan-wuest** [comment]
@poojaranjan Done. Pls review. Best regards!!

File: Merge/CommunityCall_Meeting_01.md [hunk 1]
```diff
@@ -0,0 +1,367 @@
+# Merge Community Call 1
+
+### Meeting Date/Time: Friday, Nov 5 14:00 UTC
+### Meeting Duration: 1 hour
+### [Agenda](https://github.com/ethereum/pm/issues/402)
+### [Video of the meeting](https://www.youtube.com/watch?v=_kfS9jAUY6g)
+### Moderator: T
```
Comment (poojaranjan) [inline Merge/CommunityCall_Meeting_01.md:7]:
```suggestion
### Moderator: Trent
```

**Comment by poojaranjan** [comment]
Thanks for documenting the meeting notes, @stefan-wuest!

It would be nice to have
- add a "Summary" section to capture any important points of the general agenda item
- remove "um" and "uh" 
- fix punctuations 
- break it into sentences to provide a better understanding of the context (Ref line 38, 219)

**Comment by stefan-wuest** [comment]
Thanks for the review @poojaranjan 

I will take the above points into account and adapt the notes accordingly by tomorrow night.

**Comment by stefan-wuest** [comment]
Hi @poojaranjan 
I updated it according to your input. Pls review again. Thx.

**Pull request merged**